app/auth/views.py

```diff
@@ -85,9 +85,12 @@ def login():
     if form.validate_on_submit():
         user = User.query.filter_by(email=form.email.data.lower()).first()
         print(user)
-
         if user:
-            if user.login_attempts > 2:
+            if not user.is_active :
+                current_app.logger.info('Inactive User {} trying to login'.format(current_user))
+                flash('Your account is inactive, you should contact your supervisor.', category='error')
```

Comment:
As per @MunifNagi we should just flash "Invalid username or password. Please contact HR for assistance".